content/blog/2023-04-13-rse-on-mastodon.md

```diff
@@ -0,0 +1,78 @@
++++
+title = "State of Research software engineers on Mastodon"
+
+[extra]
+
+author = "Richard Darst"
++++
+
+Many domains have migrated to Mastodon as of 2023.  Have Research
+Software Engineers?  This post examines the situation, based on [this
+post by rkdarst](https://fosstodon.org/@rkdarst/110146601969193973)
+
+Summary: not much yet
+
+This post can be updated as we get more information/the situation
+evolves, please send pull requests.
+
+
+## RSE organizations
+
+* US RSE: <https://fosstodon.org/@us_rse>
+* DE RSE: <https://fosstodon.org/@de_rse@mastodon.social>
+* Nordic RSE: <https://fosstodon.org/@nordic_rse>
+* Society of RSE (UK/int): ??
+* ... ?
+
+
+## Other accounts to follow to seed your community
+
+TODO (should we join some other list instead of make a new one here?)
+
+
+## Common hashtags
+
+* **#RSEng and #RSE** have been seen, but #RSEng is preferred
+  ([ref](https://fosstodon.org/@HeathrTurnr/110147262319789502) (same
+  as Twitter).  There doesn't seem to be much activity there.
+
+* **#HPC** has a fair community.
+
+* **#OpenScience** can be found, in addition to **ResearchData** and
+  **ResearchSoftware**.
+
+* We propose **#SciComp** for as a general "scientific computing" tag
+  which can partly overlap with the above, but is more general to
+  people interested in the computing part of science.
+
+* There is the [a.gup.pe](https://a.gup.pe/u/researchsoftware) group
+  "researchsoftware", which will boost any post tagging it, and can be
+  used as a more targeted distribution group than tags
+  ([ref](https://fosstodon.org/@mattasdata@qoto.org/110151591358979693)).
+  We could also get a listing at <https://fedi.directory/>.
+
+
+## RSE-focused Mastodon servers
+
+There don't seem to be any servers specifically focused on the RSE
+community.  Perhaps the closest one is the HPC community
+<https://mast.hpc.social>, and we are invited to take part there
+([ref](https://mast.hpc.social/@AlanSill/110147881007346377)).  Many
+of the repliers used <https://fosstodon.org>.
+
+One could ask if if's better to have a common server, or be more
+distributed among the communities of other scientists to spread the
+word.  There is a [list of academic/science related servers at
+fedi-science](https://fediscience.org/server-list.html).
+
```

Comment:
The primary advantage of having a server focused on any topic, for example RSE, is the ability to browse the local timeline without needing to wait for federation of related servers to make relevant posts local. Of course, any server can be used as a home base in the Fediverse, and any account can follow posts using a given hasttag by simply following that hashtag, so having a server for a community is largely a convenience, and doing so provides a natural gathering mechanism for people interested in a given topic.

Reply:
(added to text)